charts/argo-server/values.yaml

```diff
@@ -6,3 +6,4 @@ service:
   port: 80
   ingress:
     create: false
+debug: false
```

Comment:
I would have thought you want this to be true but I see this is just adding the ability to enable debug mode. In order to get the debug logging, would you have to change this value to true or can you pass in an argument when calling argo?

Reply:
This is the default value for the Helm chart. Users of the chart (us in monster-deploy) will need to override the value to toggle on debug mode, there's no way to reach into the templates and pass extra flags at deploy time.